README.md

```diff
@@ -30,9 +30,7 @@ func run() error {
     // The client supports multiple endpoints and will try to use one that
     // works
     client := tpb.New(
```

Comment:
There only one option, maybe we should write it on the same line ?
```go
client := tpb.New("https://apibay.org"")
```

Reply:
True, but I'll add another endpoint instead, the goal of the example is to show that it handles multiple endpoints